lib/class-wp-theme-json.php

```diff
@@ -836,53 +833,7 @@ private function get_block_styles( $style_nodes, $setting_nodes ) {
 			$node         = _wp_array_get( $this->theme_json, $metadata['path'], array() );
 			$selector     = $metadata['selector'];
 			$declarations = self::compute_style_properties( $node );
-
-			$is_link_element = self::is_link_element( $metadata['selector'] );
```

Comment:
We can remove more bytes here that we no longer need (`is_link_element`, `without_link_selector`) :hocho: :hocho: :hocho: